x/payment/client/testutil/integration_test.go

```diff
@@ -130,8 +134,19 @@ func (s *IntegrationTestSuite) TestSubmitWirePayForMessage() {
 					"test: %s, output\n:", tc.name, out.String())
 
 				events := txResp.Logs[0].GetEvents()
-				for i := 0; i < len(events); i++ {
-					s.Equal("/payment.MsgPayForMessage", events[i].GetAttributes()[0].Value)
+				for _, e := range events {
+					switch e.Type {
+					case types.EventTypePayForMessage:
+						signer := e.GetAttributes()[0].GetValue()
+						_, err = sdk.AccAddressFromBech32(signer)
+						require.NoError(err)
+						msgSize, err := strconv.ParseUint(e.GetAttributes()[1].GetValue(), 10, 64)
+						require.NoError(err)
+						s.Equal(uint64(0), msgSize%consts.ShareSize, "Message length should be multiplication of 256")
+					}
+					//s.Equal("/payment.MsgPayForMessage", events[i].GetAttributes()[0].Value)
+					//s.Equal(hexMsg, events[i].GetAttributes()[1].Value)
+					//fmt.Errorf("%d ", i) //events[i].GetAttributes()[0].Value)
```

Comment:
can we get rid of these comments if we don't need them? I think we can merge after we get rid of these!!

Reply:
@evan-forbes the code is ready to :shipit:

Reply:
could you open another PR about the msg share size?

Reply:
yep :slightly_smiling_face: 
https://github.com/celestiaorg/celestia-app/issues/245